manifests/ipset.pp

```diff
@@ -0,0 +1,62 @@
+#
+# @summary a defined resource that can match for ipsets at the top of a chain. This is a per-chain resource. You cannot mix IPv4 and IPv6 sets.
+#
+# @see http://ferm.foo-projects.org/download/2.1/ferm.html#set
+#
+# @example
+#   ferm::ipset{'CONSUL':
+#     sets => {
+#       'internet' => 'ACCEPT'
+#     },
+#   }
+#
+# @example create to matches for IPv6, both at the end of the `INPUT` chain. Explicitly mention the `filter` table.
+#   ferm::ipset{'INPUT':
+#     prepend_to_chain => false,
+#     table            => 'filter',
+#     ip_version       => 'ip6',
+#     sets               => {
+#       'testset01'      => 'ACCEPT',
+#       'anothertestset' => 'DROP'
+#     },
+#   }
+#
+# @param chain
+#   name of the chain we want to apply those rules to. The name of the defined resource will be used as default value for this.
+#
+# @param table
+#   name of the table where we want to apply  this. Defaults to `filter` because that's the most common usecase.
+#
+# @param ip_version
+#   Sadly, ip sets are version specific. You cannot mix IPv4 and IPv6 addresses. because of this, you need to provide the version.
```

Comment:
Comma after `this` would be right in German, but English only has a few use cases for commas like `, but`. Also lower case and `ipsets` for consistency.
```suggestion
sadly, ipsets are version specific. You cannot mix IPv4 and IPv6 addresses. Because of this you need to provide the version.
```

Reply:
fixed